bin/postinstall.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+SCRIPT_PATH="$(cd "$(dirname "$0")" && pwd)"
+cd "$SCRIPT_PATH"
+
+normalize()
+{
+  if [ -d $1 ]; then
+    find ./$1 -maxdepth 2 -type f -name 'normalize.css' -exec cp {} "$SCRIPT_PATH"/../generic/_generic.normalize.scss \; &&
+    exit
+  else
+    cd .. && normalize $1
+  fi
+}
+
+MODE=$1
+if [ -z "$MODE" ] ; then
+  echo -e "# manual download"
```

Comment:
At this line here, I could download all dependencies via `curl`. I know it wasn't executed by either `npm` or `bower` because no mode parameter was passed. Even a double click in the file would end up here.